app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/summary/ClaimSummaryDestination.kt

```diff
@@ -143,7 +280,7 @@ private fun PreviewClaimSummaryScreen() {
             ),
             itemType = ClaimSummaryInfoUiState.ItemType.Model(
               itemModel = ItemModel.Known(
-                displayName = "Apple iPhone 14 Pro Super Omega New Model",
+                displayName = "Apple iPhone 14 Pro",
```

Comment:
I didn't realize I left that in 😂
I did it to test longer texts 😅

Reply:
Yeah, it worked just fine with longer text, I just wanted to see how the normal-sized columns looked 😂